spec/features/user_search_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe "Filter users by firstname", type: :feature do
+
+     before(:each) do
+        @user1 = User.new(firstname:"Max", lastname:"Mustermann", username:"Maxi", email: "example@example.com")
+        @user2 = User.new(firstname:"Grace", lastname:"Hopper", username:"Profi", email: "demo@demo.com")
+        sign_in @user1
+      end
+
+    it "When users visit the user search page, then it should be possible to filter by firstname" do
```

Comment:
I think a good name would be something like: `it "should be possible to filter by firstname"

Reply:
it should form a sentence if I understood that correctly between `it` and `do`